### ISSUE TITLE
Error -61

### ISSUE THREAD

**Issue by ftsguy** [SChernykh/RandomX_OpenCL#3]
I'm getting this on my rx 550's. 
clCreateBuffer failed (c:\users\user\documents\github\randomx_opencl\randomx_opencl\miner.cpp, line 112): error -61

**Comment by SChernykh** [comment]
It's because dataset alone is more than 2 GB. You have to allocate dataset on host:
```
--dataset_host
```
in command line.

**Comment by ftsguy** [comment]
yeah i get "Couldn't allocate dataset using large pages." I tried intensity at 768 and 256 and workers at 2 as well. It locks up the pc pretty bad when i tried it.

**Comment by SChernykh** [comment]
Try intensity 384, workers at 8 and bfactor at 8. RX 550 has been tested before and did 17 h/s. RX 550 should be able to do over 100 h/s with optimized code (when it's done).

**Comment by SChernykh** [comment]
@ftsguy Can you try https://github.com/SChernykh/RandomX_OpenCL/releases/tag/1.0.4-7 ?
This command line should work for you, I expect 100-120 h/s for RX 550:
```
RandomX_OpenCL.exe --mine --validate --dataset_host --intensity 512
```
